examples/deployment/haproxy/README.md

```diff
@@ -0,0 +1,107 @@
+# Deploying HAProxy Ingress Controller
+
+Don't have a Kubernetes cluster? Single-node of [CoreOS Kubernetes](https://github.com/coreos/coreos-kubernetes/) is a good starting point.
```

Comment:
Please add this as an option here: https://github.com/kubernetes/ingress/blob/master/docs/dev/setup.md and link to it instead